documentation/book/proc-deploying-kafka-broker-kubernetes.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-broker.adoc
+
+[id='deploying-kafka-broker-kubernetes-{context}']
+= Deploying the Kafka broker to {KubernetesName}
```

Comment:
"Kafka cluster"

Reply:
@ppatierno Technically it is _Kafka and Zookeeper cluster_. Will it be clear enough that Zookeeper comes with it if we say just _Kafka cluster_?

Reply:
I might regret asking this, but why is it Kafka cluster when we're in the 'Kafka broker' chapter?

To answer Jakub, I'd hope that we can just say Kafka and not Zookeeper since we mention elsewhere that the Kafka cluster includes Zookeeper.

Reply:
Nevermind the question, @scholzj answers it later in this review :)

Reply:
Ok, if we said that Kafka Cluster includes Zoo then we should name it all Kafka Cluster.